### DISCUSSION TITLE
Export GitHub Insights Commit Graph to Slack

### DISCUSSION THREAD

**Discussion by YahelGaver** [community/community#32016] [New to GitHub]
How do I export the [graph](https://docs.github.com/en/repositories/viewing-activity-and-data-for-your-repository/analyzing-changes-to-a-repositorys-content) that show in the Commit section under the Insights tab to Slack?

**Comment by 3kh0** [comment]
Sorry but to my knowledge, you cannot export the graph to slack.

*If this helped you, please mark it as the answer!*
